packages/train2go-bridge/test/parser.test.js

```
@@ -0,0 +1,157 @@
+const { readFileSync } = require("fs");
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)